llvm/docs/DeveloperPolicy.rst

```diff
@@ -476,11 +476,12 @@ Obtaining Commit Access
 -----------------------
 
 We grant commit access to contributors with a track record of submitting high
-quality patches.  If you would like commit access, please send an email to
-`Chris <mailto:clattner@llvm.org>`_ with your GitHub username.  This is true
-for former contributors with SVN access as well as new contributors. If
-approved, a GitHub invitation will be sent to your GitHub account. In case you
-don't get notification from GitHub, go to
+quality patches.  If you would like commit access, please use this `link
+<https://github.com/llvm/llvm-project/issues/new?title=Request%20Commit%20Access%20For%20<user>&body=%23%23%23%20Why%20Are%20you%20requesting%20commit%20access%20?>`_ to file
+and issue and request commit access.  Replace the <user> string in the title
+with your github username, and explain why you are requesting commit access in
```

Comment:
I don't disagree that the explanation may be awkward, but according to @lattner's [description](https://discourse.llvm.org/t/rfc-new-criteria-for-commit-access/76290/81) of how the current process works, users are asked for a justification.

I may be overly cautious here, but I've [proposed](https://discourse.llvm.org/t/rfc-new-criteria-for-commit-access/76290) changing the commit access requirements in the past, and since this proposal was rejected,
I'm trying really hard to not change or even give the impression of changing anything about the current process, because I don't want people to think I'm trying to backdoor changes into the process when that's not my intention.

Reply:
I don't have a strong opinion on this, but it is worth mentioning that reason I brought it up is because the current documented process does not require an explanation even if that's what Chris does in practice. It *looked* like a new requirement to me because the old docs don't mention it.

That said, I'm okay keeping it as well; I totally understand not wanting it to look like backdooring changes to the process (I don't think that's what's happening here!).

Reply:
I think it’s just that when writing an email to Chris you typically wouldn’t just ask for permission and give a bit of context on who you are/why you need it, which tends to be natural in email-to-one-person format. It probably doesn’t fundamentally matter, especially if you’re asking from GitHub where it’s easy to see who you are in terms of LLVM contributions. At the same time if you offer the possibility to add some context to the user (with one of those default-text GitHub issues for example), most people would have no problem filling it up ("I have made X contributions to the project so far and I find that having commit access would help me maintain this part of the codebase more easily.")

Reply:
In practice, people include rationale for getting commit access, but it is all over the map.  It varies from:
1) Here's a number of things I've submitted.
2) My manager at (bigco) told me to apply, they are often cc'd
3) Someone told on a list told me to apply so I can land my patches.

etc.  When someone doesn't provide any rationale and there isn't anything obvious, I ask them what their plan is.